assets/js/mbta.ts

```diff
@@ -1987,16 +1987,65 @@ const stationConfig: {
   Green: {
     batchModes: { auto: true, headway: true, off: true },
     stations: [
+      {
+        id: 'GUNS',
+        name: 'Union Sq',
+        zones: {
+          e: {
+            label: "North track",
```

Comment:
@losvedir I added some labeling to GUNS and GLEC to help reflect the reality of the stations and with the aim of helping avoid confusion for users regarding the weirdness with cardinal directions along the GLX. Let me know if you have any thoughts or if you think this makes sense!

Reply:
Ideally station signage would indicate the platforms as e.g. "Track 1" and "Track 2", then we would have official names for them. @jfabi do you know if that's the case?

Reply:
Yes, **Union Square** already has the identifiers "Track 1" and "Track 2" installed as follows (photo taken from Prospect Street):

![DSC_0427](https://user-images.githubusercontent.com/3793006/150860872-0edbc5e6-e9c9-4873-82aa-11b8c06cdb15.JPG)

**At Lechmere**, the platforms are not yet signed up, but when they are installed, they will read "(D) Union Square / (E) College Ave" (on the same track as Track 1 at Union Square), and "Copley & West" (on the same track as Track 2 at Union Square).

Reply:
Thank you for the insight here @jfabi and @digitalcora , I was stuck on what to name these for a bit since they're both "westbound" tracks. Just pushed a change to reflect the correct track identifiers here and if it looks good to you @digitalcora , I will go ahead and make the corresponding change on the realtime signs end!

Reply:
Just for the record (because I was originally going off of blueprints without thinking), it should be "(D) Union Square / (E) Medford/Tufts", since College Ave station has since been renamed Medford/Tufts. (Aside: I wonder how the slashes will work there.)

This shouldn't affect this pull request as you've dropped "College Ave" entirely from the label.